src/defaults/defaults.jl

```diff
@@ -221,14 +221,15 @@ end
 ##########################################################################################
 
 function get_building_polys(buildings)
-    building_polys = fill(Point2f[], length(buildings));
-    
-    for (i, (id, b)) in enumerate(buildings)
+    building_polys = Polygon[]
+
+    for b in values(buildings)
+        polys = Vector{Point2f}[]
         for bp in b.polygons
-            building_polys[i] = begin
-                Point2f[(node.location.lon, node.location.lat) for node in bp.nodes]
-            end
+            poly = Point2f[(node.location.lon, node.location.lat) for node in bp.nodes]
+            push!(polys, poly)
         end
+        push!(building_polys, Polygon(polys[1], polys[2:end])) 
```

Comment:
Very minor suggestion: Maybe it would be a bit nicer to read if we renamed `polys` into `points` because that's what that vector holds at that moment and only gets converted into an actual `Polygon` once we push it to `building_polys`.